src/scss/pm-image-editor.scss

```diff
@@ -14,16 +14,52 @@ image-editor {
 
   image-selection {
     display: block;
-    box-sizing: border-box;
-    border: 1px solid #f00;
-    width: 100px;
-    height: 50px;
-    overflow: hidden;
+    width: 100%;
+    height: 100%;
     z-index: 10;
 
-    &[draggable] {
+    [draggable] {
       cursor: move;
     }
+
+    .image-editor-selection-border-top,
+    .image-editor-selection-border-bottom {
+        position: absolute;
+        left: 0;
+        right: 0;
+        height: 1px;
+        background-image: url('data:image/gif;base64,R0lGODlhBgABAKEAAP///wAAADY2Nv///yH/C05FVFNDQVBFMi4wAwEAAAAh/hpDcmVhdGVkIHdpdGggYWpheGxvYWQuaW5mbwAh+QQACgD/ACwAAAAABgABAAACAxQuUgAh+QQBCgADACwAAAAABgABAAACA5SAUgAh+QQBCgADACwAAAAABgABAAACA5SBBQAh+QQBCgADACwAAAAABgABAAACA4QOUAAh+QQBCgADACwAAAAABgABAAACAwSEUAAh+QQBCgADACwAAAAABgABAAACA4SFBQA7');
+        z-index: 1;
+    }
+    .image-editor-selection-border-top {
+        top: 0;
+    }
+    .image-editor-selection-border-bottom {
+        bottom: 0
+    }
+    .image-editor-selection-border-left,
+    .image-editor-selection-border-right {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        width: 1px;
+        background-image: url('data:image/gif;base64,R0lGODlhAQAGAKEAAP///wAAADY2Nv///yH/C05FVFNDQVBFMi4wAwEAAAAh/hpDcmVhdGVkIHdpdGggYWpheGxvYWQuaW5mbwAh+QQACgD/ACwAAAAAAQAGAAACAxQuUgAh+QQBCgADACwAAAAAAQAGAAACA5SAUgAh+QQBCgADACwAAAAAAQAGAAACA5SBBQAh+QQBCgADACwAAAAAAQAGAAACA4QOUAAh+QQBCgADACwAAAAAAQAGAAACAwSEUAAh+QQBCgADACwAAAAAAQAGAAACA4SFBQA7');
```

Comment:
Can you comment where the source image file is for this? So if we need to update the icon, we know where to look.